### PR TITLE
Add Rowe Morehouse

### PR DESCRIPTION
**Pull request by rowe-morehouse**

### CODE REVIEW

File: src/data.js [hunk 1]
```diff
@@ -1,5 +1,29 @@
 // keep it commonjs export
 module.exports = [
+  {
+    name: 'Rowe Morehouse',
+    description:
+      'Growth · Product · Software Project Management · Frontend Dev · Design · Technical Writing · Sales',
+    url: 'https://rowe-morehouse.github.io/resume/uses/',
```
Comment (davidak) [inline src/data.js:7]: the page don't shows what tools you use, so it don't fit's this projecs

**Review by davidak — CHANGES_REQUESTED** [review]

File: src/data.js (hunk 1 above)
Comment (rowe-morehouse) [reply src/data.js:7]: Look again, the page has all the tools I use: [https://rowe-morehouse.github.io/resume/uses/](https://rowe-morehouse.github.io/resume/uses/)

Comment (davidak) [reply src/data.js:7]: nice

**Review by davidak — APPROVED** [review]
LGTM

**Review by HugoDF — APPROVED** [review]

**Pull request merged**